apps/dev-portal-web/domains/docs/components/AmountDistributionCalculator.tsx

```diff
@@ -0,0 +1,357 @@
+import { faker } from '@faker-js/faker'
+import { Col, Row, RowProps } from 'antd'
+import React, { useCallback, useEffect, useState } from 'react'
+import { useIntl } from 'react-intl'
+
+import { Plus, Trash } from '@open-condo/icons'
+import { Alert, Button, Card, Input, List, MarkdownCodeWrapper, Switch, Tabs, Typography } from '@open-condo/ui'
+
+import { split, createRecipientKey } from '@condo/domains/acquiring/utils/billingCentrifuge'
```

Comment:
Seems wrong...I guess its time to migrate to packages

Reply:
Will migrate to a separate package soon

Reply:
Moved to open-condo/billing